tests/baselines/reference/jsDeclarationsGetterSetter.js

```diff
@@ -106,19 +122,22 @@ export class B {
     /**
      * @param {number} _arg
      */
-    set x(arg: number);
+    set x(_arg: number);
 }
 export class C {
-    set x(arg: number);
+    set x(_arg: number);
     get x(): number;
 }
 export class D {
     get x(): number;
 }
 export class E {
-    set x(arg: number);
+    set x(_arg: number);
 }
 export class F {
-    set x(arg: number);
+    set x(_arg: number);
     get x(): number;
 }
+export class G {
+    set x(args: number);

```

Comment:
this could be generated as `args_0` (or smth like that) - but I don't think it's worth the added code complexity